vm/src/lib.rs

```diff
@@ -85,6 +85,8 @@ pub use rustpython_common as common;
 #[cfg(feature = "rustpython-compiler")]
 pub use rustpython_compiler as compile;
 
+use crate::stdlib::errno::errors;
```

Comment:
why is this module imported here? I expect this is imported from `os_error_optional_new`.

Reply:
My mistake.